src/components/Features/CultureSelector.vue

```diff
@@ -1,8 +1,34 @@
 <template>
-  <fieldset class="fr-input-group" v-for="(culture) in uuidedCultures" :key="culture.id">
-    <label class="fr-label">Type de culture</label>
+  <fieldset class="culture-group fr-mb-1w fr-p-3w" v-for="(culture) in uuidedCultures" :key="culture.id">
+    <div class="fr-input-group">
+      <label class="fr-label" :for="`cpf-${culture.id}-input`">Type de culture</label>
```

Comment:
`autocomplete` suffixe systématiquement avec un `-input`, d'où le `id` et le `for` légèrement différents.